### PR TITLE
DOC: update the Period.daysinmonth docstring

### PR DESCRIPTION
**Pull request by IHackPy**
# Period.daysinmonth
################################################################################
################################################################################

Return total number of days of the month in given Period.

This attribute returns the total number of days of given month

Returns
-------
Int
Number of days with in month

See also
--------
Period.dayofweek
    Return the day of the week

Period.dayofyear
    Return the day of the year

Examples
--------
>>> import pandas as pd
>>> p = pd.Period("2018-03-11", freq='H')
>>> p.daysinmonth
31

################################################################################
################################################################################

Docstring for "pandas.Period.daysinmonth" correct. :)

### CODE REVIEW

**Comment by WillAyd** [comment]
xref #20294 a lot of the feedback will be similar

**Comment by HimanshuAwasthi95** [comment]
sure will work on both Thaks for reviewing

File: pandas/_libs/tslibs/period.pyx [hunk 1]
```diff
@@ -1275,6 +1275,25 @@ cdef class _Period(object):
 
     @property
     def daysinmonth(self):
+        """
+        Get total days of month that a Period falls on.
```
Comment (jorisvandenbossche) [inline pandas/_libs/tslibs/period.pyx]: I would maybe rephrase this as "Get the total number of days of the month that the Period falls in"

File: pandas/_libs/tslibs/period.pyx [hunk 2]
```diff
@@ -1275,6 +1275,25 @@ cdef class _Period(object):
...
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.days_in_month : Return the days of the month
+
```
Comment (jorisvandenbossche) [inline pandas/_libs/tslibs/period.pyx]: no blank line is needed here

**Review by jorisvandenbossche — COMMENTED** [review]
Small comment, looks good for the rest

File: pandas/_libs/tslibs/period.pyx (hunk 1 above)
Comment (HimanshuAwasthi95) [reply pandas/_libs/tslibs/period.pyx]: Sure will update now . As I'm not good in making sentences thanks for guiding :) :+1:

**Comment by HimanshuAwasthi95** [comment]
@jorisvandenbossche why ci test fail here ?

**Comment by jreback** [comment]
ignore the ci

i have had to cancel most jobs

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@HimanshuAwasthi95 Thanks!

**Comment by HimanshuAwasthi95** [comment]
thank u @jorisvandenbossche @WillAyd @jreback
